activemq-broker/src/main/java/org/apache/activemq/broker/jmx/PersistenceAdapterView.java

```diff
@@ -64,11 +88,46 @@ private String invoke(Callable<String> callable) {
         return result;
     }
 
+    private String serializePersistenceAdapterStatistics() {
+        if (persistenceAdapterStatistics != null) {
+            try {
+                final ObjectMapper mapper = new ObjectMapper();
```

Comment:
ObjectMapper is threadsafe so you should just create it one time for the class so we don't need to re-create it over and over.

Reply:
Good catch!